src/Institution/Sparql/SparqlRepository.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Institution\Sparql;
+
+use App\Institution\Institution;
+use App\Institution\InstitutionRepository;
+use App\Ontology\Org;
+use App\Rdf\Client;
+use App\Rdf\Iri;
+use App\Rdf\SparqlQueryBuilder;
+
+final class SparqlRepository implements InstitutionRepository
+{
+    /**
+     * @var Client
+     */
+    private $rdfClient;
+
+    public function __construct(
+        Client $rdfClient
+    ) {
+        $this->rdfClient = $rdfClient;
+    }
+
+    /**
+     * @param int $offset
+     * @param int $limit
+     *
+     * @return Institution[]
+     */
+    public function all(int $offset = 0, int $limit = 100): array
```

Comment:
all is probably an incorrect naming, perhaps better to call it fetch as it's always limited, for an all method I would probably expect it to return an iterator to loop over all records.

Reply:
Can't we just use the method names from https://www.doctrine-project.org/api/orm/latest/Doctrine/ORM/EntityRepository.html ?

Reply:
Agree, all is a bit strange here. @benhillier I don't think that we should stick to ORM repository. 
Our API (Data types, Interfaces, etc...) should come out of our needs. 

Anyway `fetch(int $limit, int $offset)` or `find(SomeCustomQueryType $query)`  looks better.